packages/terra-doc-template/src/ExampleTemplate.scss

```diff
@@ -0,0 +1,73 @@
+:local {
+  .template {
+    border: 1px solid 	#e8e8e8;
+    border-radius: 3px;
+    overflow: hidden;
+  }
+
+  .template + .template {
+    margin-top: 0.9375rem;
+  }
+
+  .header {
+    border-bottom: 1px solid #e8e8e8;
+    padding: 0.625rem;
+  }
+
+  .content {
+    padding: 0.625rem;
+  }
+
+  .description {
+    padding-bottom: 0.625rem;
+  }
+
+  .footer {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+  }
+
+  .code {
+    height: 0;
+  }
+
+  .is-expanded {
+    height: auto;
+  }
+
+  .toggle {
+    align-items: center;
+    border: 0;
+    cursor: pointer;
+    display: flex;
+    margin-left: auto;
+    outline: none;
+    padding: 0.625rem;
+  }
+
+  .chevron-left {
+    background: url('data:image/svg+xml;base64,PHN2ZyBkYXRhLW5hbWU9IkxheWVyIDEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2%0D%0AZyIgdmlld0JveD0iMCAwIDQ4IDQ4IiBjbGFzcz0iaXMtYmlkaSI+PHBhdGggZD0iTTEwLjMgMjRM%0D%0AMzMuOCAwbDMuOSAzLjhMMTggMjRsMTkuNyAyMC4yLTMuOSAzLjh6Ii8+PC9zdmc+');
```

Comment:
Should we use chevrons from `terra-icon`? Also, we can use `inline-svg` from our mixins here.

Reply:
This is using the icons from terra-icon, but in CSS to avoid adding an additional dependency to the package. Because this component is becoming a hard dependency in every package I think we should avoid adding any extra dependencies if there is an alternative solution. 

The inline-svg from the mixins converts a raw SVG into the base64 encode. The SVG here is already encoded. Unless there are additional mutations the mixin performs I think this should behave the exact same way, and doesn't require adding the terra-mixin dependency. 

@bjankord Do you see any issues not using the inline-svg mixin if the SVG is already encoded?

Reply:
No concern, the mixin is a nicety so people don't need to convert to dataURI, if its already converted, the mixin isn't needed.